Sources/Xcodeproj/pbxproj().swift

```diff
@@ -0,0 +1,184 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+
+//TODO escaping
+
+
+import PackageType
+import Utility
+
+public func pbxproj(package package: Package, modules: [SwiftModule], products _: [Product], printer print: (String) -> Void) {
+
+    let srcroot = package.path
+    let nontests = modules.filter{ !($0 is TestModule) }
+    let tests = modules.filter{ $0 is TestModule }
```

Comment:
Swift: the two year old language where apparently we already have extremely firm guidelines for whitespace.